icons/pin.svg

```diff
@@ -0,0 +1,13 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M9 3v5l-2 4v2h10M9 3h6M9 3H7m10 11h-5v7m5-7v-2l-2-4V3m0 0h2" />
```

Comment:
Can you split this path up in smaller parts. Otherwise design programs like figma don't like it.

Reply:
sure

Reply:
like this? Please check the most recent commit.